.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+Resolves: [issue_number]
```

Comment:
I'm not sure if it's required by GH might need the name of the file in UPPER_CASE